### ISSUE TITLE
word_word

### ISSUE THREAD

**Issue by CigaLi** [yao8839836/text_gcn#37]

**Comment by CigaLi** [comment]
hello，I run the model and word_word edges and word_document edges use **random weight**,but i still get results similar to the original results of the articleg，which means the weight is not important?
Is the structure of the graph a decisive factor?

**Comment by yao8839836** [comment]
@CigaLi 

Yes, the structure of a graph (if an edge exists or not) is more important than the edge weight. But I found TF-IDF can perform better than TF, and PMI performs better than word co-occurrence count, which means edge weights also matters.

Thanks.
